app/Providers/FoundationServiceProvider.php

```diff
@@ -71,11 +71,11 @@ public function registerTranslations()
     protected function registerConfig()
     {
         $this->publishes([
-            DASHBOARD_PATH . '/resources/stubs/config/content.php' => config_path('content.php'),
+            DASHBOARD_PATH.'/resources/stubs/config/content.php' => config_path('content.php'),
         ]);
 
         $this->mergeConfigFrom(
-            DASHBOARD_PATH . '/resources/stubs/config/content.php', 'content'
+            DASHBOARD_PATH.'/resources/stubs/config/content.php', 'content'
```

Comment:
__[code_sniffer]__
* Only one argument is allowed per line in a multi-line function call

<!-- Comment by [SideCI](https://sideci.com) -->